qa/httpCalls/transaction/findTransactionByID.py

```diff
@@ -1,14 +1,16 @@
 import json
 
-#executes a  transaction/findById call
-def http_transaction_findById(sidechainNode, txid):
-      j = {\
-            "transactionId": txid, \
-            "format": True \
-      }
-      request = json.dumps(j)
-      response = sidechainNode.transaction_findById(request)
-      return response["result"]
 
+# executes a  transaction/findById call
+def http_transaction_findById(sidechainNode, txid):
+    j = { \
+        "transactionId": txid, \
+        "format": True \
```

Comment:
unnecessary backslashes, why do these exist? :thinking:

Reply:
Either that was there before, or the formatter added them. I don't recall writing them 😺 .

Reply:
They were there before yes. Always leave code cleaner than it was before :stuck_out_tongue_winking_eye: